src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -113,6 +113,18 @@ source %{
     return vt->length();
   }
 
+  static inline uint vector_length_in_bytes(const Node* n) {
```

Comment:
Is it possible that we reuse the existed functions `vector_length` ? Besides, the argument type is  `Node*` which is not the same with other functions that use `MachNode*` .  Use the same type?

Reply:
We port these functions from x86.ad.  We will change the type to `MachNode *` in next commit. Thank you.